### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by Abraham12611**
Closes: #10
Added contents from the link to the newly created file.

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!-- < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < ☺
+v                               ✰  Thanks for creating a PR! ✰    
+v    If your PR doesn't close an issue, that's OK!  Just remove the Closes: #XXX line!
+☺ > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > >  -->
+
+Closes: #10
```
Comment (tungbq) [inline pull_request_template.md:6]:
```suggestion
Closes: #XXX
```

**Review by tungbq — CHANGES_REQUESTED** [review]
@Abraham12611 thanks for your change, please place the file under` .github ` folder: ` .github/pull_request_template.md`

**Comment by Abraham12611** [comment]
Made the changes.

File: pull_request_template.md
Comment (tungbq) [inline pull_request_template.md:1]: Please remove this file

Comment (tungbq) [reply pull_request_template.md:1]: No worries. I've just pushed a commit to update it

**Pull request merged**